aiapy/calibrate/util.py

```diff
@@ -183,8 +268,14 @@ def get_pointing_table(start, end):
 
     Parameters
     ----------
-    start : `~astropy.time.Time`
-    end : `~astropy.time.Time`
+    source : str
+        Name of the source from which to retrieve the pointing table.
+        Must be one of ``"jsoc"`` or ``"lmsal"``.
+        Note that the LMSAL pointing table is not updated frequently.
```

Comment:
It would be helpful to explain more here what the difference is between these two or to do it in the main body in the docstring above.

Reply:
Done